mantidproject/mantid
# Correct Units when plotting IvsQ curve normalised by bin-width

It was found that when plotting a workspace that was normalised by the bin-widths (in A^-1 for example) then the y-axis unit would be labelled as (A^-1-1). This was technically correctly as we are dividing by the x-axis unit. However in the format A^-1-1 it seems to read more as A^-2 instead of (A^-1)^-1. As a fix, an extra set of parenthesize have been provided to clear up any ambiguity.

**To Test**
- Make sure that your data directories are able to search the data archive
- run this script:

``` python
from mantid.simpleapi import *
from numpy import sin, pi

Load(Filename='INTER00013463.nxs', OutputWorkspace='INTER00013463')
dref = mtd['INTER00013463']

IvsQ, IvsLam, thetaOut=ReflectometryReductionOneAuto(dref,AnalysisMode="MultiDetectorAnalysis",ProcessingInstructions="4", ThetaIn="0.5037",
    WavelengthMin=2.2,CorrectionAlgorithm='None',StrictSpectrumChecking=False,CorrectDetectorPositions=True)
```
- Plot the IvsQ workspace
- Make sure that the y-units are of the format ([x-unit])^-1

**Release Notes**: //to be updated

Fixes #13513
